chain/chain-primitives/src/error.rs

```diff
@@ -221,15 +217,14 @@ impl Error {
         match self.kind() {
             ErrorKind::Unfit(_)
             | ErrorKind::Orphan
-            | ErrorKind::BlockMissing(_)
             | ErrorKind::ChunkMissing(_)
             | ErrorKind::ChunksMissing(_)
             | ErrorKind::InvalidChunkHeight
             | ErrorKind::IOErr(_)
             | ErrorKind::Other(_)
             | ErrorKind::ValidatorError(_)
             // TODO: can be either way?
```

Comment:
can we remove this TODO

Reply:
It doesn't look like there is any action item here -- we cannot attribute epoch out of bound to malicious activities.

Reply:
Done.